eslint.config.js

```diff
@@ -4,8 +4,17 @@ import svelte_config from '@sveltejs/eslint-config';
 export default [
 	...svelte_config,
 	{
+		languageOptions: {
+			parserOptions: {
+				project: true
+			}
+		},
 		rules: {
+			'@typescript-eslint/await-thenable': 'error',
+//			'@typescript-eslint/no-floating-promises': 'error',
+//			'@typescript-eslint/no-misused-promises': 'error',
```

Comment:
I took a look at the errors that arise if this rule is enabled, and not a single one is valid. It's just ESLint whining pointlessly

```suggestion
```

Reply:
a lot more errors pop up with `no-floating-promises` and i haven't looked at them all yet, but of the ones i _have_ looked at it's the same story. definitely not something we want

Reply:
I don't understand why the issues `no-floating-promises` is identifying shouldn't be addressed. I just pushed a commit addressing a lot of them: https://github.com/sveltejs/kit/pull/12284/commits/743c640971c20515f69bd4b27e155176c0e834fc. We can drop it if they're really not issues. This isn't my strong point, so it's quite possible I'm missing something. I figured I'd push it though so that I can at least learn what that is as it's easier to discuss that way.

Reply:
They're really not issues, but the rule tricked you into breaking code that works perfectly well